community-code/README.md

```diff
@@ -11,6 +11,7 @@
 | Title | Blog | Code | Neptune
 | --- | :---: | :---: | :---:
 | MLOps For Time Series Prediction: Binance Trading Tutorial | [![blog]](https://neptune.ai/blog/mlops-pipeline-for-time-series-prediction-tutorial) | [![github]](./binance-trading-neptune-master) | [![neptune]](https://app.neptune.ai/o/community/org/mlops-pipeline-for-time-series-prediction/runs/table?viewId=standard-view)
+| How to build a RAG system using LangChain | [![blog]]() | [![github]](./HOW_TO_BUILD_A_RAG_SYSTEM_USING_LANGCHAIN/) | [![neptune]](https://app.neptune.ai/o/community/org/building-RAG-using-LangChain/runs/table?viewId=standard-view)
```

Comment:
Please add the blog link

Reply:
It's currently under review. Maybe that's something we can ask your marketing team (Dominika) about when it's ready on their end?

Reply:
Here's the URL that'll be used: https://neptune.ai/blog/building-rag-system-using-langchain